draft-ietf-quic-ack-frequency.md

```diff
@@ -162,6 +162,11 @@ receiving ACK_FREQUENCY frames.  If an endpoint sends the transport parameter,
 the peer is allowed to send ACK_FREQUENCY frames independent of whether it also
 sends the min_ack_delay transport parameter or not.
 
+An endpoint MUST NOT send ACK_FREQUENCY frames until it has received the
+min_ack_delay transport parameter from the peer.  When clients use 0-RTT, they
+MUST NOT store the value of the server's min_ack_delay transport parameter.
```

Comment:
How about going like:
```suggestion
Endpoints MUST NOT remember the value of min_ack_delay transport parameter they received.
```

I think provision of the first sentence ("An endpoint MUST NOT send ACK_FREQUENCY frames until ...") is covered by the last sentence of the previous paragraph. If we think that that sentence is vague, we can change "if" of that sentence to "if and only if."

Regarding the second sentence, my preference goes to using the phrase "remembered" and talking about both sides, because that is what RFC 9000 does.

Reply:
Thanks, suggestion accepted.